presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/environment/SinglenodePostgresql.java

```diff
@@ -39,14 +39,14 @@
     // Use non-default PostgreSQL port to avoid conflicts with locally installed PostgreSQL if any.
     public static final int POSTGRESQL_PORT = 15432;
 
-    private final DockerFiles dockerFiles;
+    private final DockerFiles.ResourceProvider resourceProvider;
```

Comment:
can you extract adding `resourceProvider` to separate commit?

Reply:
Yup, done